### ISSUE TITLE
Hello, may I ask on which dimension the node features are concatenated when using 'cat'?

### ISSUE THREAD

**Issue by xiaobingbuhuitou** [GGchen1997/STEPS_Bioinformatics#3]
`node_feat0 = torch.cat([lm_embedding.squeeze(0)[1:-1, :], node_feat], dim=1)` 大佬请问在节点特征和序列特征进行cat时是在哪一个维度上进行cat的，lm_embedding形状为[B L hid_dim]，这样是在 长度L维度上拼接，不应该在特征维度上拼接么（这里B为1么？所以压缩了？）？因为没有示例数据所以这是我猜测的，恳请您能讲解。谢谢！

**Comment by GGchen1997** [comment]
We concat in the feature dimension.
